config/default/manager_webhook_patch.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        ports:
+        - containerPort: 9443
+          name: webhook-server
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /tmp/k8s-webhook-server/serving-certs
```

Comment:
it is right to use `/tmp/` here?

Reply:
The patch is used in conjunction with cert-manager which is disabled. Created like this by default.